### ISSUE TITLE
JS runtime for Bun

### ISSUE THREAD

**Issue by UrNightmaree** [webui-dev/webui#267]
Will there be a support for running JS/TS via [Bun](https://bun.sh)?

**Comment by hassandraga** [comment]
Yes, we are planing to support Bun. We need first to finalize the [Deno Wrapper](https://github.com/webui-dev/deno-webui), then creating a new [Bun Wrapper](https://github.com/webui-dev/bun-webui) by converting Deno APIs in [webui.ts](https://github.com/webui-dev/deno-webui/blob/main/src/webui.ts) to Bun APIs.

**Comment by jinzhongjia** [comment]
I started to adapt bunjs. You can refer to this issue https://github.com/webui-dev/bun-webui/issues/1 for the specific progress.

**Comment by komothecat** [comment]
> I started to adapt bunjs. You can refer to this issue [webui-dev/bun-webui#1](https://github.com/webui-dev/bun-webui/issues/1) for the specific progress.

Wonderful! I'll keep tracks of any updates. Thanks for working on it.
